### PR TITLE
Add valueLink into valueButton binding

### PR DESCRIPTION
**Pull request by galachad**
For improve the functionality of `valueButton` add `valueLink`. Same as in shiny.

### CODE REVIEW

File: R/valueButton.R [hunk 1]
```diff
@@ -51,4 +48,30 @@ valueButton <- function(inputId, label, selector, attribute = "value", icon = NU
...
+      `data-val` = NULL,
+      `data-value-attribute` = attribute,
+      `data-selector` = selector,
+      list(icon, label),
+      ...
+    )
+  )
+}
+
+value_button_dependency <- function() {
```
Comment (krystian8207) [inline R/valueButton.R:70]: Do we need to wrap this into function?

File: R/valueButton.R [hunk 2]
```diff
@@ -28,15 +28,12 @@
...
 #' @export
 valueButton <- function(inputId, label, selector, attribute = "value", icon = NULL, width = NULL, ...) {
 
   shiny::tagList(
-    shiny::singleton(
-      shiny::tags$head(
-        shiny::tags$script(type = "text/javascript", src = "shinyGizmo/valuebutton.js")
-      )
-    ),
+    value_button_dependency,
```
Comment (krystian8207) [inline R/valueButton.R:36]: Please make a function call if `value_button_dependency` is a function.

**Review by krystian8207 — CHANGES_REQUESTED** [review]
LGTM, please address a few comments.
More to that:
- please add an example gif/screenshot in the readme,
- bump package version and update NEWS.md file.

File: R/valueButton.R (hunk 1 above)
Comment (galachad) [reply R/valueButton.R:70]: Do you think it should be a static object?

**Pull request merged**